os/include/tinyara/reboot_reason.h

```diff
@@ -28,8 +28,8 @@ typedef enum {
 	 * System : 50 ~ 79
 	 * Network : 80 ~ 109,
 	 * Common Service : 110 ~ 139,
-	 * App : 140 ~ 249
-	 * Board Specific : 250 ~ 254
+	 * App : 140 ~ 246
```

Comment:
Sorry but, let's split commit for Tizen Lite change and board change.
This file should be a separate commit.

Reply:
added separate commit